cobalt/profiler/profiler_frame.idl

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Quick drive by comment, we usually put a comment with a link to the spec in the IDL files, see https://chromium.googlesource.com/chromium/src/+/main/third_party/blink/renderer/core/timing/profiler_frame.idl for an example, or other IDL files in Cobalt repo.

Reply:
Done.